build/ci/Dockerfile.unittest

```diff
@@ -25,6 +25,12 @@ RUN go get github.com/onsi/ginkgo/ginkgo@v1.16.4 \
     && make kustomize \
     && make controller-gen
 
+ARG YQ_VERSION=v4.13.5
+ARG ARCH=yq_linux_386
+RUN ARCH=$(if [ "$(uname -m)" = "x86_64" ]; then echo "amd64"; else echo "$(uname -m)" ; fi) \
+    && wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_${ARCH} -O /usr/bin/yq \ 
```

Comment:
@matancarmeli7 you can use go get - https://github.com/mikefarah/yq#go-get

Reply:
done [here](https://github.com/IBM/ibm-block-csi-operator/blob/afcb83e53488bd038841e4ac2bc3302865dbce8e/build/ci/Dockerfile.unittest#L25)